merkato/merkato.py

```diff
@@ -113,6 +115,12 @@ def distribute_bids(self, total_to_distribute, step):
         #    order placed in decaying_bid_ladder
         pass
 
+    def newTransactionHistory(self, newTransactionHistory):
```

Comment:
we might want to add this info to db to allow fancy queries. @15chrjef @livinginformation